<launch>
  <!-- a basic listener node -->
  <node name="keyboard" pkg="keyboard" type="keyboard">
  	<!-- <remap from="keyboard/keydown" to="rotors1/keyboard/keydown"/> -->
  </node>
<!--   <node name="keyboard" pkg="keyboard" type="keyboard">
  	<remap from="keyboard/keydown" to="rotors2/keyboard/keydown"/>
  </node>
 -->
  <!-- pass args to the listener node -->
  <node name="px4_offboard_position_control" pkg="px4_offboard_position_control" type="px4_offboard_position_control" output="screen">
  	<param name="uav_num" value="2" />  	
  </node>

<!--   <node name="px4_offboard_position_control" pkg="px4_offboard_position_control" type="px4_offboard_position_control" output="screen">
  	<param name="uav_id" value="2" />  	
  </node>
   -->
</launch>

